src/components/Editor/EditorReducer.js

```
@@ -0,0 +1,85 @@
+                tabs: [...state.tabs, action.payload]
+            };
+
+        case "SET_PARENT_TAB_GROUP_ID":
+            return {
+                ...state,
+                parentTabGroupId: action.payload
+            };
+
+
+        case "SELECT_TAB":
+            const tab = state.tabs.find(obj => obj.id === action.payload);
```

Biome (2.4.7)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)